README.md

```diff
@@ -20,18 +20,16 @@ $ wget https://github.com/conda-forge/miniforge/releases/latest/download/Minifor
 # For MacOS:
 # $ wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-x86_64.sh
 
+# Install Miniforge/conda:
 $ bash ./Miniforge3-Linux-x86_64.sh
-# make sure the installer runs 'conda init' at the end of installation, or run it manually.
-
-# Restart shell to load conda integration
-$ exec bash
 
 # Optional: create conda environment
-$ conda create -n dgfem
-$ conda activate dgfem
+$ export MY_CONDA=/path/to/installed/conda # Default installation path: $HOME/miniforge3
+$ $MY_CONDA/bin/conda create -n dgfem
+$ . $MY_CONDA/bin/activate dgfem
```

Comment:
There should be a brief explainer somewhere in here on how to re-activate the env after installation.

Reply:
I added a comment below this.